example/tutorials/tutorial_grafana.md

```diff
@@ -33,7 +33,13 @@ Depending on your KubeArmor setup, you can follow:
     --set loki.storage.type="filesystem" \
     --set loki.auth_enabled=false \
     --set singleBinary.replicas=1
+
+    ```
+    If integration with Alertmanager is required, along with retention period, we can use `../loki-values.yaml` and run:
+    ```bash
+    helm upgrade --install loki grafana/loki --values ../loki-values.yaml
```

Comment:
Can we have a small reference to Alert Manager installation guide so as if people want to install it they can be redirected there?